City-of-Helsinki/drupal-helfi-etusivu
# UHF-5665: Styles fixes:

# Current -paragraph (https://helsinkisolutionoffice.atlassian.net/browse/UHF-5665)

## What was done?
- Fix linter errors
- Remove irrelevant :before styles
- Add offset to illustration on breakpoint-m
- Use unified hdbt style for highlight links
- Remove alt text + translations from illustrative image

## How to install
- Checkout branch
- Run `composer require drupal/hdbt:dev-UHF-5664-unify-highlight-links`
- Run `drush cr` 

## How to test
- Add a `Current` -paragraph and a `Popular services` -paragraph to a landing page
- Check that the look matches the frontpage sketch 
- Check scss code

## Designers review

One of the checkboxes below needs to be checked (`[x]`)

* [x] This PR has been visually reviewed by a designer (Sami)

## Other PRs
https://github.com/City-of-Helsinki/drupal-hdbt/pull/341
